.github/workflows/ipt.yml

```diff
@@ -0,0 +1,65 @@
+name: IPT Build/Release
+
+on:
+  push:
+    # takes muliple branch names
+    branches: 
+      - main
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*' # 2021.01.x
+    tags:
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9][0-9]' # 2021.01.01
```

Comment:
We actually don't need this to build when pushing to a numbered tag since we'll be triggering that manually.